CODE_REVIEW.md

```diff
@@ -0,0 +1,23 @@
+# Task 1: Code fixes and review
+
+#### 1.1 Code smells & improvements
```

Comment:
There is a problem with `BookSearchComponent` in the `ngOnInit` method.  Can you describe what the problem is, and how you would solve it?

Hint: `ReadingListComponent` doesn't have the same issue.

Reply:
Thanks, I will have a look at this one.

Reply:
I updated the code to consume the observable directly vs ngOnInit, as we shouldn't mutate a store observable like that.